### DISCUSSION TITLE
pypi update when?

### DISCUSSION THREAD

**Discussion by GDjkhp** [Xtr4F/PyCharacterAI#7] [General]
i wanna just use "pip install PyCharacterAI" because im lazy

**Comment by Xtr4F** [comment]
I'm actively working on updates right now and am constantly adding/changing things. When I feel that the library is stable, I will upload it to pypi. It's already pretty stable tho, I just want to add a few more things.

**Comment by Xtr4F** [comment]
The project is now available on PyPI again!
